chart/openfaas/values.yaml

```diff
@@ -180,6 +182,10 @@ nats:
 # ingress configuration
 ingress:
   enabled: false
+  ## For k8s >= 1.18 you need to specify the pathType
+  ## See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#better-path-matching-with-path-types
+  pathType: ImplementationSpecific
```

Comment:
@alexellis : Should we enable pathType or comment it out?

Reply:
I am not sure that it's required for this part of the chart, so let's go with less or more. It's easily reversible.

Reply:
ok commented it out